packages/react-ui-core/src/index.js

```diff
@@ -87,3 +87,8 @@ export {
 export {
   Highlighter,
 } from './Highlighter'
+
+export {
+  ToggleButton,
+} from './ToggleButton'
+
```

Comment:
Wouldn't you export ToggleButton from Button?

Reply:
This was a suggestion from tad
"Please export like everything else is exported:

export { default } from './ToggleButton'
export { default as ToggleButton } from './ToggleButton'
Also, this probably belongs in react-ui-core/src/Button directory."